### PR TITLE
Initial project skeleton structure.

### PR DESCRIPTION
**Pull request by tomikazi**

### CODE REVIEW

File: pkg/northbound/server.go [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+	lis, err := net.Listen("tcp", fmt.Sprintf(":%d", s.cfg.Port))
+	if err != nil {
+		return err
+	}
+
+	tlsCfg := &tls.Config{}
+
+	if *s.cfg.CertPath == "" && *s.cfg.KeyPath == "" {
+		// Load default Certificates
+		clientCerts, err := tls.X509KeyPair([]byte(certs.DefaultLocalhostCrt), []byte(certs.DefaultLocalhostKey))
```
Comment (SeanCondon) [inline pkg/northbound/server.go]: Is it bringing over default-certificates.go? This should be be certs.DefaultClientCrt and certs.DefaultClientKey

Comment (tomikazi) [reply pkg/northbound/server.go]: Hmmm. I followed pattern from onos-config. Is onos-config also wrong then?

File: pkg/northbound/proto/admin.proto [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+limitations under the License.
+*/
+
+syntax = "proto3";
+
+// Package admin defines the administrative and diagnostic gRPC interfaces.
+package proto;
+
+// ZTPAdminService provides means for enhanced interactions with the zero-touch-provisioning subsystem.
+service ZTPAdminService {
```
[the remark this answers is not in the document]
Comment (tomikazi) [reply pkg/northbound/proto/admin.proto]: Done

File: cmd/onos-ztp/onos-ztp.go [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+	// Sync the glog and klog flags.
+	flag.CommandLine.VisitAll(func(f1 *flag.Flag) {
+		f2 := klogFlags.Lookup(f1.Name)
+		if f2 != nil {
+			value := f1.Value.String()
+			_ = f2.Value.Set(value)
+		}
+	})
+	log.Info("Starting onos-control")
```
[the remark this answers is not in the document]
Comment (tomikazi) [reply cmd/onos-ztp/onos-ztp.go]: Done.

**Pull request merged**